gammapy/irf/tests/test_effective_area.py

```diff
@@ -130,3 +130,13 @@ def test_EffectiveAreaTable2D(method):
     actual = effarea.evaluate(offset=offset, energy=energy).shape
     desired = np.zeros([nbinso, nbinse]).shape
     assert_equal(actual, desired)
+
+    #Test ARF export
```

Comment:
The test that writes ARF files should be for the `EffectiveAreaTable` class.
There you could write it, read it back into a second object and assert that some things in it's content are the same before and after writing.

Here you should just test `to_effective_area`. You could e.g. assert that evaluating the 1D and 2D effective area gives identical or close results if you use the same offset.